website/docs/r/container_node_pool.html.markdown

```diff
@@ -52,7 +52,42 @@ resource "google_container_cluster" "primary" {
   }
 }
 ```
+### Usage with an empty default pool.
+```hcl
+resource "google_container_node_pool" "np" {
+  name               = "my-node-pool"
+  zone               = "us-central1-a"
+  cluster            = "${google_container_cluster.primary.name}"
+  node_count = 1
+  
+  node_config {
+    preemptible = true
+    machine_type = "n1-standard-1"
+    oauth_scopes = [
+      "compute-rw",
+      "storage-ro",
+      "logging-write",
+      "monitoring", 
+    ]
+  }
+}
+
+resource "google_container_cluster" "primary" {
+  name               = "marcellus-wallace"
+  zone               = "us-central1-a"
 
+  lifecycle {
+    ignore_changes = [
+      "node_pool"
+    ]
+  }
+  
+  node_pool = [{
```

Comment:
This can just be:
```
node_pool {
  name = "default-pool"
}
```

Reply:
Fixed.

Reply:
The `= [` is also unnecessary

Reply:
Indeed my bad,i kinda misread that.
reFixed :)